<!DOCTYPE html><html lang="en-US"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><link rel="stylesheet" href="/assets/css/just-the-docs-default.css"><link rel="stylesheet" href="/assets/css/just-the-docs-head-nav.css" id="jtd-head-nav-stylesheet"><style id="jtd-nav-activation"> .site-nav > ul.nav-list:first-child > li:not(:nth-child(4)) > a, .site-nav > ul.nav-list:first-child > li > ul > li a { background-image: none; } .site-nav > ul.nav-list:not(:first-child) a, .site-nav li.external a { background-image: none; } .site-nav > ul.nav-list:first-child > li:nth-child(4) > a { font-weight: 600; text-decoration: none; }.site-nav > ul.nav-list:first-child > li:nth-child(4) > button svg { transform: rotate(-90deg); }.site-nav > ul.nav-list:first-child > li.nav-list-item:nth-child(4) > ul.nav-list { display: block; }</style><script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script><meta name="viewport" content="width=device-width, initial-scale=1"><link rel="icon" href="/favicon.ico" type="image/x-icon"><title>Knowledge Base | IEI genetics</title><meta name="generator" content="Jekyll v4.4.1" /><meta property="og:title" content="Knowledge Base" /><meta property="og:locale" content="en_US" /><meta name="description" content="Diagnosis of IEI and PID" /><meta property="og:description" content="Diagnosis of IEI and PID" /><link rel="canonical" href="http://localhost:4000/docs/knowledge_base/" /><meta property="og:url" content="http://localhost:4000/docs/knowledge_base/" /><meta property="og:site_name" content="IEI genetics" /><meta property="og:type" content="website" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Knowledge Base" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"WebPage","description":"Diagnosis of IEI and PID","headline":"Knowledge Base","url":"http://localhost:4000/docs/knowledge_base/"}</script><body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" class="d-none"> <symbol id="svg-link" viewBox="0 0 24 24"><title>Link</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"><title>Menu</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"><line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"><title>Expand</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"><polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"><title id="svg-external-link-title">(external link)</title><path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"><title>Document</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"><path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"><title>Search</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-copy" viewBox="0 0 16 16"><title>Copy</title><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard" viewBox="0 0 16 16"><path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"/><path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"/> </svg> </symbol> <symbol id="svg-copied" viewBox="0 0 16 16"><title>Copied</title><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard-check-fill" viewBox="0 0 16 16"><path d="M6.5 0A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3Zm3 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3Z"/><path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1A2.5 2.5 0 0 1 9.5 5h-3A2.5 2.5 0 0 1 4 2.5v-1Zm6.854 7.354-3 3a.5.5 0 0 1-.708 0l-1.5-1.5a.5.5 0 0 1 .708-.708L7.5 10.793l2.646-2.647a.5.5 0 0 1 .708.708Z"/> </svg> </symbol> </svg><div class="side-bar"><div class="site-header" role="banner"> <a href="/" class="site-title lh-tight"> IEI genetics </a> <button id="menu-button" class="site-button btn-reset" aria-label="Toggle menu" aria-pressed="false"> <svg viewBox="0 0 24 24" class="icon" aria-hidden="true"><use xlink:href="#svg-menu"></use></svg> </button></div><nav aria-label="Main" id="site-nav" class="site-nav"><ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a><li class="nav-list-item"><a href="/docs/data_cleaning/" class="nav-list-link">Data cleaning</a><li class="nav-list-item"><a href="/docs/iuis_iei_datasource/" class="nav-list-link">IUIS IEI data source</a><li class="nav-list-item"><a href="/docs/knowledge_base/" class="nav-list-link">Knowledge Base</a><li class="nav-list-item"><a href="/docs/user_experience/" class="nav-list-link">User experience</a></ul><ul class="nav-list"><li class="nav-list-item external"> <a href="https://github.com/iei-genetics/iei-genetics.github.io" class="nav-list-link external" > IEI-genetics on GitHub <svg viewBox="0 0 24 24" aria-labelledby="svg-external-link-title"><use xlink:href="#svg-external-link"></use></svg> </a></ul></nav><footer class="site-footer"> This site runs from GitHub <a href="https://github.com/iei-genetics/">iei-genetics</a>.</footer></div><div class="main" id="top"><div id="main-header" class="main-header"><div class="search" role="search"><div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search IEI genetics" aria-label="Search IEI genetics" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label></div><div id="search-results" class="search-results"></div></div><nav aria-label="Auxiliary" class="aux-nav"><ul class="aux-nav-list"><li class="aux-nav-list-item"> <a href="https://github.com/iei-genetics/iei-genetics.github.io" class="site-button" > IEI-genetics on GitHub </a></ul></nav></div><div class="main-content-wrap"><div id="main-content" class="main-content"><main><h1 id="knowledge-base-linked-resources"> <a href="#knowledge-base-linked-resources" class="anchor-heading" aria-labelledby="knowledge-base-linked-resources"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Knowledge Base Linked Resources</h1><h2 id="gnomad"> <a href="#gnomad" class="anchor-heading" aria-labelledby="gnomad"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> GnomAD</h2><p>GnomAD is a large-scale resource that aggregates and harmonises sequencing data from diverse cohorts to provide a comprehensive view of human genetic variation. We used the v4 dataset which comprises genomics data from 807,162 individuals. Beyond its extensive catalogue of variant calls, including over 786 million single nucleotide variants and 122 million InDels, the database offers detailed annotations that support analyses of population-specific allele frequencies.</p><p><strong>Citations:</strong></p><ul><li>Karczewski KJ, Francioli LC, Tiao G, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature. 2020;581(7809):434-443. <a href="https://doi.org/10.1038/s41586-020-2308-7">doi:10.1038/s41586-020-2308-7</a></ul><h2 id="omim---online-mendelian-inheritance-in-man"> <a href="#omim---online-mendelian-inheritance-in-man" class="anchor-heading" aria-labelledby="omim---online-mendelian-inheritance-in-man"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> OMIM - Online Mendelian Inheritance in Man</h2><p>An online catalogue of human genes and genetic disorders, OMIM provides comprehensive information on gene-disease relationships, genetic mechanisms, and phenotype descriptions. Curated by the McKusick-Nathans Institute of Genetic Medicine at Johns Hopkins University, this resource is essential for clinicians, researchers, and geneticists. An Online Catalog of Human Genes and Genetic Disorders. Online Mendelian Inheritance in Man, OMIM. McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University (Baltimore, MD) <a href="https://omim.org/">https://omim.org/</a>.</p><h2 id="alphafold-and-alphamissense"> <a href="#alphafold-and-alphamissense" class="anchor-heading" aria-labelledby="alphafold-and-alphamissense"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> AlphaFold and AlphaMissense</h2><p>We link to access detailed data for your gene - including information on gene structure, function with <a href="https://www.alphafold.ebi.ac.uk">AlphaFold</a> and <a href="https://alphamissense.hegelab.org">AlphaMissense</a> pathogenicity predictions. This page provides a view:</p><ul><li><strong>Gene Details:</strong> Basic gene data and annotations.<li><strong>Protein Structure:</strong> 3D coordinates, per-residue confidence (pLDDT) and Predicted Aligned Error (PAE) for assessing domain packing.<li><strong>Function:</strong> Annotations on protein function and regions of interest.<li><strong>AlphaMissense Predictions:</strong> AI-driven scoring categorising missense mutations as likely pathogenic, likely benign or ambiguous, helping to highlight potentially disease-causing variants.</ul><p><strong>Citations:</strong></p><ul><li>Jumper, J., Evans, R., Pritzel, A. et al. <em>Highly accurate protein structure prediction with AlphaFold.</em> Nature 596, 583–589 (2021). <a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a><li>Jun Cheng et al. <em>Accurate proteome-wide missense variant effect prediction with AlphaMissense.</em> Science 381, eadg7492 (2023). <a href="https://doi.org/10.1126/science.adg7492">https://doi.org/10.1126/science.adg7492</a></ul><h2 id="clinvar-a-public-archive-of-human-variation"> <a href="#clinvar-a-public-archive-of-human-variation" class="anchor-heading" aria-labelledby="clinvar-a-public-archive-of-human-variation"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> ClinVar: A Public Archive of Human Variation</h2><p>ClinVar provides detailed classifications (e.g., “Pathogenic,” “Likely pathogenic,” “Benign,” etc.) along with supporting evidence and review status. It aggregates submissions from multiple sources to present consensus and conflicting interpretations, maps variants to reference sequences per HGVS standards, and collaborates with expert panels like ClinGen for continual re-evaluation. Data is accessible via the website, FTP, and APIs for diverse clinical and research applications.</p><p><strong>Citations:</strong></p><ul><li>Landrum MJ, Chitipiralla S, Kaur K, et al. <em>ClinVar: updates to support classifications of both germline and somatic variants.</em> Nucleic Acids Res. 2024 Nov 23:gkae1090. doi: <a href="https://doi.org/10.1093/nar/gkae1090">10.1093/nar/gkae1090</a>.</ul><h2 id="dbnsfp"> <a href="#dbnsfp" class="anchor-heading" aria-labelledby="dbnsfp"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> dbNSFP</h2><p>For the research and testing phase we also used dbNSFP, a comprehensive database designed for the functional prediction and annotation of all potential non-synonymous single-nucleotide variants (nsSNVs) and splicing-site SNVs in human protein-coding genes. It includes over 120 million variant entries and aggregates prediction scores from 33 different sources and allele frequencies from major population datasets.</p><p><strong>Citations:</strong></p><ul><li>Xiaoming Liu, Chang Li, Chengcheng Mou, Yibo Dong, and Yicheng Tu. dbNSFP v4: a comprehensive database of transcript-specific functional pre- dictions and annotations for human nonsynonymous and splice-site SNVs. Genome Medicine, 12(1):103, December 2020. ISSN 1756-994X. <a href="https://doi.org/10.1186/s13073-020-00803-9">doi:10.1186/s13073-020-00803-9</a></ul></main></div></div><div class="search-overlay"></div></div><script src="https://cdn.jsdelivr.net/npm/mermaid@9.1.6/dist/mermaid.min.js"></script> <script> var config = {} ; mermaid.initialize(config); window.mermaid.init(undefined, document.querySelectorAll('.language-mermaid')); </script>
