<!DOCTYPE html><html lang="en-US"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><link rel="stylesheet" href="/assets/css/just-the-docs-default.css"><link rel="stylesheet" href="/assets/css/just-the-docs-head-nav.css" id="jtd-head-nav-stylesheet"><style id="jtd-nav-activation"> .site-nav > ul.nav-list:first-child > li:not(:nth-child(3)) > a, .site-nav > ul.nav-list:first-child > li > ul > li a { background-image: none; } .site-nav > ul.nav-list:not(:first-child) a, .site-nav li.external a { background-image: none; } .site-nav > ul.nav-list:first-child > li:nth-child(3) > a { font-weight: 600; text-decoration: none; }.site-nav > ul.nav-list:first-child > li:nth-child(3) > button svg { transform: rotate(-90deg); }.site-nav > ul.nav-list:first-child > li.nav-list-item:nth-child(3) > ul.nav-list { display: block; }</style><script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script><meta name="viewport" content="width=device-width, initial-scale=1"><link rel="icon" href="/favicon.ico" type="image/x-icon"><title>Definitions | IEI genetics</title><meta name="generator" content="Jekyll v4.4.1" /><meta property="og:title" content="Definitions" /><meta property="og:locale" content="en_US" /><meta name="description" content="Diagnosis of IEI and PID" /><meta property="og:description" content="Diagnosis of IEI and PID" /><link rel="canonical" href="http://localhost:4000/docs/definitions/" /><meta property="og:url" content="http://localhost:4000/docs/definitions/" /><meta property="og:site_name" content="IEI genetics" /><meta property="og:type" content="website" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Definitions" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"WebPage","description":"Diagnosis of IEI and PID","headline":"Definitions","url":"http://localhost:4000/docs/definitions/"}</script><body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" class="d-none"> <symbol id="svg-link" viewBox="0 0 24 24"><title>Link</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"><title>Menu</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"><line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"><title>Expand</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"><polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"><title id="svg-external-link-title">(external link)</title><path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"><title>Document</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"><path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"><title>Search</title><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-copy" viewBox="0 0 16 16"><title>Copy</title><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard" viewBox="0 0 16 16"><path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"/><path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"/> </svg> </symbol> <symbol id="svg-copied" viewBox="0 0 16 16"><title>Copied</title><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard-check-fill" viewBox="0 0 16 16"><path d="M6.5 0A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3Zm3 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3Z"/><path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1A2.5 2.5 0 0 1 9.5 5h-3A2.5 2.5 0 0 1 4 2.5v-1Zm6.854 7.354-3 3a.5.5 0 0 1-.708 0l-1.5-1.5a.5.5 0 0 1 .708-.708L7.5 10.793l2.646-2.647a.5.5 0 0 1 .708.708Z"/> </svg> </symbol> </svg><div class="side-bar"><div class="site-header" role="banner"> <a href="/" class="site-title lh-tight"> IEI genetics </a> <button id="menu-button" class="site-button btn-reset" aria-label="Toggle menu" aria-pressed="false"> <svg viewBox="0 0 24 24" class="icon" aria-hidden="true"><use xlink:href="#svg-menu"></use></svg> </button></div><nav aria-label="Main" id="site-nav" class="site-nav"><ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a><li class="nav-list-item"><a href="/docs/data_cleaning/" class="nav-list-link">Data cleaning</a><li class="nav-list-item"><a href="/docs/definitions/" class="nav-list-link">Definitions</a><li class="nav-list-item"><a href="/docs/iuis_iei_datasource/" class="nav-list-link">IUIS IEI data source</a><li class="nav-list-item"><a href="/docs/knowledge_base/" class="nav-list-link">Knowledge Base</a><li class="nav-list-item"><a href="/docs/user_experience/" class="nav-list-link">User experience</a></ul><ul class="nav-list"><li class="nav-list-item external"> <a href="https://github.com/iei-genetics/iei-genetics.github.io" class="nav-list-link external" > IEI-genetics on GitHub <svg viewBox="0 0 24 24" aria-labelledby="svg-external-link-title"><use xlink:href="#svg-external-link"></use></svg> </a></ul></nav><footer class="site-footer"> This site runs from GitHub <a href="https://github.com/iei-genetics/">iei-genetics</a>.</footer></div><div class="main" id="top"><div id="main-header" class="main-header"><div class="search" role="search"><div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search IEI genetics" aria-label="Search IEI genetics" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label></div><div id="search-results" class="search-results"></div></div><nav aria-label="Auxiliary" class="aux-nav"><ul class="aux-nav-list"><li class="aux-nav-list-item"> <a href="https://github.com/iei-genetics/iei-genetics.github.io" class="site-button" > IEI-genetics on GitHub </a></ul></nav></div><div class="main-content-wrap"><div id="main-content" class="main-content"><main><h1 id="definitions"> <a href="#definitions" class="anchor-heading" aria-labelledby="definitions"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Definitions</h1><p>Last update: 20250329</p><p>All data is available in the downloadable table including those marked as hidden.</p><hr /><h3 id="major-category"> <a href="#major-category" class="anchor-heading" aria-labelledby="major-category"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Major category</h3><ul><li><strong>Description:</strong> A high-level classification that groups genes based on their overall role in immune function or disease.<li><strong>Notes:</strong> Uses a custom mapping (<code class="language-plaintext highlighter-rouge">major_map</code>) to convert source categories into standardised group names.</ul><h3 id="subcategory"> <a href="#subcategory" class="anchor-heading" aria-labelledby="subcategory"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Subcategory</h3><ul><li><strong>Description:</strong> A more detailed classification within the major category, providing finer granularity about the gene’s role in immunity or disease.</ul><h3 id="disease"> <a href="#disease" class="anchor-heading" aria-labelledby="disease"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Disease</h3><ul><li><strong>Description:</strong> Indicates the type or classification of disease associated with the gene, reflecting the clinical context.</ul><h3 id="genetic-defect"> <a href="#genetic-defect" class="anchor-heading" aria-labelledby="genetic-defect"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Genetic defect</h3><ul><li><strong>Description:</strong> Identifies the gene or specific genetic variant responsible for the condition.<li><strong>Interaction:</strong> Typically links to external databases (e.g. ClinVar, OMIM) for more detailed information.</ul><h3 id="inheritance"> <a href="#inheritance" class="anchor-heading" aria-labelledby="inheritance"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Inheritance</h3><ul><li><strong>Description:</strong> The mode of inheritance for the condition associated with the gene.<li><strong>Possible Values:</strong><ul><li><strong>AD:</strong> Autosomal Dominant<li><strong>AR:</strong> Autosomal Recessive<li><strong>XL / XLR:</strong> X-linked (Recessive)<li><strong>AD/AR:</strong> Combination of autosomal dominant and recessive patterns<li><strong>Sporadic:</strong> Non-familial or toxin-induced cases</ul><li><strong>Visual Cues:</strong> Displayed with colour coding to distinguish between different modes.</ul><h3 id="icd9"> <a href="#icd9" class="anchor-heading" aria-labelledby="icd9"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> ICD9</h3><ul><li><strong>Description:</strong> International Classification of Diseases, Ninth Revision code.</ul><h3 id="icd10"> <a href="#icd10" class="anchor-heading" aria-labelledby="icd10"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> ICD10</h3><ul><li><strong>Description:</strong> International Classification of Diseases, Tenth Revision code.</ul><h3 id="omim-id"> <a href="#omim-id" class="anchor-heading" aria-labelledby="omim-id"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> OMIM ID</h3><ul><li><strong>Description:</strong> The Online Mendelian Inheritance in Man identifier, providing access to detailed genetic disorder information.</ul><h3 id="hpo-ids"> <a href="#hpo-ids" class="anchor-heading" aria-labelledby="hpo-ids"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> HPO IDs</h3><ul><li><strong>Description:</strong> Combined Human Phenotype Ontology identifiers relevant to the gene.</ul><h3 id="hpo-term"> <a href="#hpo-term" class="anchor-heading" aria-labelledby="hpo-term"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> HPO term</h3><ul><li><strong>Description:</strong> Specific Human Phenotype Ontology term describing a clinical feature.</ul><h3 id="associated-features"> <a href="#associated-features" class="anchor-heading" aria-labelledby="associated-features"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Associated features</h3><ul><li><strong>Description:</strong> Other clinical or phenotypic features associated with the disease.</ul><h3 id="uniprot"> <a href="#uniprot" class="anchor-heading" aria-labelledby="uniprot"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Uniprot</h3><ul><li><strong>Description:</strong> The UniProt accession ID for the gene’s protein, with a link to protein structure and function data (e.g. from AlphaFold or Alpha Missense).</ul><h3 id="alphafold_url-hidden"> <a href="#alphafold_url-hidden" class="anchor-heading" aria-labelledby="alphafold_url-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> AlphaFold_URL (hidden)</h3><ul><li><strong>Description:</strong> A URL linking to the AlphaFold database entry. This provides access to variant and protein structure predictions for the gene.</ul><h3 id="score_positive_total"> <a href="#score_positive_total" class="anchor-heading" aria-labelledby="score_positive_total"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score_positive_total</h3><ul><li><strong>Description:</strong> The overall gene score computed from ClinVar variant reports.<li><strong>Calculation:</strong> Sums the scores from variants considered pathogenic or likely pathogenic (ignoring incidental benign variants).<li><strong>Interpretation:</strong> A higher score indicates a greater burden of pathogenic variants.</ul><h3 id="probabilities"> <a href="#probabilities" class="anchor-heading" aria-labelledby="probabilities"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> probabilities</h3><ul><li><strong>Description:</strong> A sparkline boxplot summarising the distribution of the –log10(prior probability) for pathogenic variants for each gene. See the Variant Risk Estimate for detailed calculations.<li><strong>Visualisation:</strong> Uses the pre-calculated minimum, first quartile, median, third quartile, and maximum values to provide a compact view of the probability distribution.<li><strong>Usage:</strong> Helps assess how common a candidate pathogenic variant might be in patients.</ul><h3 id="variantcountsvre"> <a href="#variantcountsvre" class="anchor-heading" aria-labelledby="variantcountsvre"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> VariantCounts.VRE</h3><ul><li><strong>Description:</strong> A visual summary of ClinVar single nucleotide variant (SNV) classifications for the gene. See the Variant Risk Estimate for detailed calculations.<li><strong>Visualisation:</strong> Displayed as a sparkline bar chart with:<ul><li><strong>Red:</strong> Variants with score <strong>5</strong> (Pathogenic)<li><strong>Orange:</strong> Variants with score <strong>2-4</strong> (Likely pathogenic)<li><strong>Light blue:</strong> Variants with score <strong>1-2</strong> (Uncertain/conflicting)<li><strong>Dark blue:</strong> Variants with score <strong>0</strong> (Benign)</ul><li><strong>Interaction:</strong> Clicking the chart links to a ClinVar search for more details.</ul><h3 id="variantcountsclnvar"> <a href="#variantcountsclnvar" class="anchor-heading" aria-labelledby="variantcountsclnvar"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> VariantCounts.ClnVar</h3><ul><li><strong>Description:</strong> A composite visual summary of all ClinVar variant reports for the gene. This data is the “raw” version directly from ClinVar which contains many overlapping descriptions for single variants, transcripts, or genes.<li><strong>Visualisation:</strong> Displayed as a sparkline bar chart with:<ul><li><strong>Red:</strong> Variants with score <strong>5</strong> (Pathogenic)<li><strong>Orange:</strong> Variants with score <strong>3-4</strong> (Likely pathogenic)<li><strong>Light blue:</strong> Variants with score <strong>1-2</strong> (Uncertain/conflicting)<li><strong>Dark blue:</strong> Variants with score <strong>0</strong> (Benign)</ul><li><strong>Interaction:</strong> Clicking the chart links to a ClinVar search for the gene.</ul><h3 id="b-cell"> <a href="#b-cell" class="anchor-heading" aria-labelledby="b-cell"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> B cell</h3><ul><li><strong>Description:</strong> Status of B cell counts (e.g. normal, low, decreased, immature).</ul><h3 id="t-cell"> <a href="#t-cell" class="anchor-heading" aria-labelledby="t-cell"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> T cell</h3><ul><li><strong>Description:</strong> Status of T cell counts (e.g. low, normal, high, defective).</ul><h3 id="ig"> <a href="#ig" class="anchor-heading" aria-labelledby="ig"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Ig</h3><ul><li><strong>Description:</strong> Immunoglobulin (Ig) levels or status.</ul><h3 id="neutrophil"> <a href="#neutrophil" class="anchor-heading" aria-labelledby="neutrophil"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Neutrophil</h3><ul><li><strong>Description:</strong> Neutrophil count status.</ul><h3 id="inheritance-detail"> <a href="#inheritance-detail" class="anchor-heading" aria-labelledby="inheritance-detail"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Inheritance detail</h3><ul><li><strong>Description:</strong> Further detailed description of the mode of inheritance.</ul><h3 id="major_category_original-hidden"> <a href="#major_category_original-hidden" class="anchor-heading" aria-labelledby="major_category_original-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Major_category_original (hidden)</h3><ul><li><strong>Description:</strong> The original category label as provided in the source data before any recoding or mapping.</ul><h3 id="gofdn-details"> <a href="#gofdn-details" class="anchor-heading" aria-labelledby="gofdn-details"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> GOF/DN details</h3><ul><li><strong>Description:</strong> Details on gain-of-function or dominant-negative effects observed for the gene, as per the orginal IUIS IEI source.</ul><h3 id="t-cell-details"> <a href="#t-cell-details" class="anchor-heading" aria-labelledby="t-cell-details"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> T cell details</h3><ul><li><strong>Description:</strong> Extra details regarding T cell abnormalities, as per the orginal IUIS IEI source.</ul><h3 id="b-cell-details"> <a href="#b-cell-details" class="anchor-heading" aria-labelledby="b-cell-details"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> B cell details</h3><ul><li><strong>Description:</strong> Extra details regarding B cell abnormalities, as per the orginal IUIS IEI source.</ul><h3 id="ig-details"> <a href="#ig-details" class="anchor-heading" aria-labelledby="ig-details"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Ig details</h3><ul><li><strong>Description:</strong> Additional information regarding immunoglobulin levels, as per the orginal IUIS IEI source.</ul><h3 id="neutrophil-details"> <a href="#neutrophil-details" class="anchor-heading" aria-labelledby="neutrophil-details"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Neutrophil details</h3><ul><li><strong>Description:</strong> Additional details regarding neutrophil counts, as per the orginal IUIS IEI source.</ul><h3 id="other-affected-cells"> <a href="#other-affected-cells" class="anchor-heading" aria-labelledby="other-affected-cells"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Other affected cells</h3><ul><li><strong>Description:</strong> Other immune or non-immune cells that may be impacted, as per the orginal IUIS IEI source.</ul><h3 id="score5clnvar-hidden"> <a href="#score5clnvar-hidden" class="anchor-heading" aria-labelledby="score5clnvar-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score5.ClnVar (hidden)</h3><ul><li><strong>Description:</strong> Count of ClinVar variant reports with a score of <strong>5</strong> (Pathogenic).</ul><h3 id="score4clnvar-hidden"> <a href="#score4clnvar-hidden" class="anchor-heading" aria-labelledby="score4clnvar-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score4.ClnVar (hidden)</h3><ul><li><strong>Description:</strong> Count of ClinVar variant reports with a score of <strong>3-4</strong> (Likely pathogenic).</ul><h3 id="score2clnvar-hidden"> <a href="#score2clnvar-hidden" class="anchor-heading" aria-labelledby="score2clnvar-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score2.ClnVar (hidden)</h3><ul><li><strong>Description:</strong> Count of ClinVar variant reports with a score of <strong>1-2</strong> (typically representing uncertain or conflicting classifications).</ul><h3 id="score0clnvar-hidden"> <a href="#score0clnvar-hidden" class="anchor-heading" aria-labelledby="score0clnvar-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score0.ClnVar (hidden)</h3><ul><li><strong>Description:</strong> Count of ClinVar variant reports with a score of <strong>0</strong> (representing benign variants with scores in the range -5 to -1).</ul><h3 id="score5vre-hidden"> <a href="#score5vre-hidden" class="anchor-heading" aria-labelledby="score5vre-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score5.VRE (hidden)</h3><ul><li><strong>Description:</strong> Count of SNVs (single nucleotide variants) with a score of <strong>5</strong> (Pathogenic) as per the Variant Risk Estimation.</ul><h3 id="score4vre-hidden"> <a href="#score4vre-hidden" class="anchor-heading" aria-labelledby="score4vre-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score4.VRE (hidden)</h3><ul><li><strong>Description:</strong> Count of SNVs with a score of <strong>2-4</strong> (Likely pathogenic) as per the Variant Risk Estimation.</ul><h3 id="score2vre-hidden"> <a href="#score2vre-hidden" class="anchor-heading" aria-labelledby="score2vre-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score2.VRE (hidden)</h3><ul><li><strong>Description:</strong> Count of SNVs with a score of <strong>1-2</strong> (Uncertain or conflicting) as per the Variant Risk Estimation.</ul><h3 id="score0vre-hidden"> <a href="#score0vre-hidden" class="anchor-heading" aria-labelledby="score0vre-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> score0.VRE (hidden)</h3><ul><li><strong>Description:</strong> Count of SNVs with a score of <strong>0</strong> (Benign) as per the Variant Risk Estimation.</ul><h3 id="min_prob-hidden"> <a href="#min_prob-hidden" class="anchor-heading" aria-labelledby="min_prob-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> min_prob (hidden)</h3><ul><li><strong>Description:</strong> The minimum prior probability (–log10) observed for the gene.</ul><h3 id="median_prob-hidden"> <a href="#median_prob-hidden" class="anchor-heading" aria-labelledby="median_prob-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> median_prob (hidden)</h3><ul><li><strong>Description:</strong> The median prior probability (–log10) observed for the gene.</ul><h3 id="max_prob-hidden"> <a href="#max_prob-hidden" class="anchor-heading" aria-labelledby="max_prob-hidden"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> max_prob (hidden)</h3><ul><li><strong>Description:</strong> The maximum prior probability (–log10) observed for the gene.</ul></main></div></div><div class="search-overlay"></div></div><script src="https://cdn.jsdelivr.net/npm/mermaid@9.1.6/dist/mermaid.min.js"></script> <script> var config = {} ; mermaid.initialize(config); window.mermaid.init(undefined, document.querySelectorAll('.language-mermaid')); </script>
